liquibase-core/src/main/java/liquibase/sqlgenerator/core/CreateTableGenerator.java

```diff
@@ -340,7 +340,7 @@ protected Relation getAffectedTable(CreateTableStatement statement) {
         return new Table().setName(statement.getTableName()).setSchema(new Schema(statement.getCatalogName(), statement.getSchemaName()));
     }
 
-    private boolean constraintNameAfterUnique(Database database) {
+    private boolean isInformixDatabase(Database database) {
```

Comment:
The old function name was more informative. We care that the constraintName comes after the unique clause. It happens to be only for informix databases currently, but in the future it might not be.